src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -28,6 +30,71 @@ public SarifValidatorTests()
             _schema = SchemaReader.ReadSchema(schemaText, JsonSchemaFile);
         }
 
+        //[Fact]
+        internal void ValidateAllTheThings()
+        {
+            // First, we start with builders that only populate required properties that are backed by primitives.
```

Comment:
>// [](start = 12, length = 2)

Comment is inconsistent with the name of the test and with the call to `GetBuildersForAllPrimitives`. Copy/paste error in comment?